package.json

```diff
@@ -33,7 +32,7 @@
         "vuex-class": "^0.3.2"
     },
     "devDependencies": {
-        "@nuxt/bridge": "^3.1.0",
+        "@nuxt/bridge": "npm:@nuxt/bridge-edge@latest",
```

Comment:
What does this do? always Installs the latest version of that dependency?

Reply:
Yes, it gets the latest version of nuxt bridge.